### PR TITLE
feat(flags): add uk flag

### PR DESCRIPTION
**Pull request by sachinrameshn**
### Requirements
Add UK flag to the ui-kit

### Description of the Change

The UK flag has been added to the UI Kit.

Fix#: MANAGER-10212

### CODE REVIEW

**Review by stif59100 — APPROVED** [review]

File: packages/base/core/src/fonts/oui-icons/oui-icons.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @font-face {
     font-family: "oui-icons";
-    src: url("./oui-icons.woff2?e6207891c6a5d3a86e0fd46c2cd72256") format("woff2"),
-url("./oui-icons.woff?e6207891c6a5d3a86e0fd46c2cd72256") format("woff");
+    src: url("./oui-icons.woff2?1b0e6bb125f6f2a02c3b6a16808cdcd7") format("woff2"),
+url("./oui-icons.woff?1b0e6bb125f6f2a02c3b6a16808cdcd7") format("woff");
```
Comment (mohammed-zahaf) [inline packages/base/core/src/fonts/oui-icons/oui-icons.css:4]: question, how do you get this `1b0e6bb125f6f2a02c3b6a16808cdcd7` ?

Comment (sachinrameshn) [reply packages/base/core/src/fonts/oui-icons/oui-icons.css:4]: Yarn flag will re generates this.

Comment (sachinrameshn) [reply packages/base/core/src/fonts/oui-icons/oui-icons.css:4]: Command: yarn flag

Comment (mohammed-zahaf) [reply packages/base/core/src/fonts/oui-icons/oui-icons.css:4]: Ah got it, thanks

**Review by mohammed-zahaf — APPROVED** [review]

**Comment by antleblanc** [comment]
This flag already exist.
see: https://github.com/ovh/ovh-ui-kit/blob/master/packages/base/core/src/svg/flags/gb.svg 

**question:** Couldn't existing flag being used instead of add a duplicate?

**Comment by sachinrameshn** [comment]
> This flag already exist. see: https://github.com/ovh/ovh-ui-kit/blob/master/packages/base/core/src/svg/flags/gb.svg
> 
> **question:** Couldn't existing flag being used instead of add a duplicate?

Helllo Antoine, Api is returning "uk" as value and we have "gb" defined in UI kit. Hence manager is not able to fetch uk flag from the UI kit.

**Comment by sachinrameshn** [comment]
Closing this PR as we are doing the changes in Manager app based on discussion with Service delivery and stack team.

**Pull request closed without merging**